server/connectors/api-connector-sqlite/src/main/scala/com/prisma/api/connector/sqlite/SQLiteDatabasesFactory.scala

```diff
@@ -7,7 +7,6 @@ import slick.jdbc.SQLiteProfile
 import slick.jdbc.SQLiteProfile.api._
 
 object SQLiteDatabasesFactory {
-  private lazy val dbDriver = new org.sqlite.JDBC
```

Comment:
All other database factories hand in the JDBC driver as an argument instead of hardcoding it. I don't know why that is exactly. I think @dpetrick introduced that structure. Can you talk to him to understand why?